### DISCUSSION TITLE
Microsoft's PS Module 'ReportingServicesTools' uses same Module GUID as dbatools

### DISCUSSION THREAD

**Discussion by fm-knopfler** [dataplat/dbatools#9236] [General]
I've noticed that 'ReportingServicesTools' uses the same **unique** identifier for it's module as dbatools. Checking PSGallery, it seems dbatools was first to use it. I've filed an issue in the project but haven't received a reaction so far.

Maybe someone can bring it to their attention.

microsoft/ReportingServicesTools#421

**Comment by wsmelton** [comment]
Once the module has been published to the PS Gallery with that GUID you can't change it. That I'm aware it is literally on place it matters so do not expect the tools team that owns that RS module to change it.

**Comment by potatoqualitee** [comment]
man that is crazy, i never noticed this or thought it'd work
